hail/src/main/scala/is/hail/expr/ir/EmitClassBuilder.scala

```diff
@@ -1032,6 +1045,16 @@ class EmitMethodBuilder[C](
             mb.getArg(codeIndex + i)(t).get
           }))
       }
+
+      override def get(cb: EmitCodeBuilder): PValue = {
```

Comment:
This seems wrong

Reply:
Agreed, clearly wrong. I didn't expect anyone to look at this as soon as I posted haha, I usually go through the diff myself while the tests are running and clean up anything I forgot about. Clearly I didn't finish implementing `get` on the different `EmitValue`s.

Reply:
Fixed, I think.